### DISCUSSION TITLE
Одновременная работа на нескольких компьютерах

### DISCUSSION THREAD

**Discussion by Vacym** [Pavlik-g/library_project#12] [Q&A] [answered]
Как насчет того, чтобы сделать возможность подключаться нескольким компьютерам по локальной сети и одновременно редактировать одну и ту же базу данных?
Насколько это будет сложно реализовать и есть ли смысл реализовывать такое вообще?

**Comment by Djacon** [comment, marked as answer]
1. Можно, но я этого делать не буду
2. Смысл реализации есть, но не для нашей школы
